README.md

```diff
@@ -25,6 +25,13 @@ Dataset is consisted of **38** disease classes from [PlantVillage](https://plant
  **Go to http://localhost:5000/ to test your app.**
 
 ## Server Set-Up:
+<br>**GCP**      - Intermediate</br>
+<br>**Gradient** - Easy</br>
+<br>**AWS EC2**  - Advanced</br>
```

Comment:
Please don't use <br> before GCP, Gradient, and AWS EC2. And you forgot to mention link!

Reply:
single br tag after intermediate and easy will work.